### PR TITLE
Add CLA workflow

### PR DESCRIPTION
**Pull request by nicolasstucki**
Based on https://github.com/scala-js/scala-js/blob/master/ci/check-cla.sh

### CODE REVIEW

**Comment by nicolasstucki** [comment]
I have no idea where the older version of the CLA check is setup. We still need to disable that one. (odd `CLA`, new `Scala CLA`)

**Review by liufengyun — APPROVED** [review]
LGTM

I couldn't find the old one either.

**Comment by sjrd** [comment]
Just make sure that `check-cla.sh` has the executable bit set in git. I don't know how to check that on the GitHub UI. Otherwise LGTM.

**Comment by smarter** [comment]
I tweaked the settings so the original CLA check isn't required anymore to merge something, I'll try disabling the scala-webapps.epfl.ch webhook to see if that also removes it from the list of checks we get.

File: project/scripts/check-cla.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+set -eux
+
+AUTHOR=$GITHUB_ACTOR
+echo "Pull request submitted by $AUTHOR";
+signed=$(curl -s https://www.lightbend.com/contribute/cla/scala/check/$AUTHOR | jq -r ".signed");
+if [ "$signed" = "true" ] ; then
+  echo "CLA check for $AUTHOR successful";
+else
+  echo "CLA check for $AUTHOR failed";
+  echo "Please sign the Scala CLA to contribute to Scala.js.";
```
Comment (smarter) [inline project/scripts/check-cla.sh:11]:
```suggestion
  echo "Please sign the Scala CLA to contribute to the Scala compiler";
```

**Pull request merged**